index.js

```diff
@@ -2,17 +2,20 @@ import { Activity } from './lib/activity.js'
 import { Logs } from './lib/log.js'
 import { Metrics } from './lib/metrics.js'
 import { getPaths, getDefaultRootDirs } from './lib/paths.js'
+import fs from 'node:fs/promises'
+import { join } from 'node:path'
 
 export { ActivityEvent } from './lib/activity.js'
 export { MetricsEvent } from './lib/metrics.js'
 
 export class Core {
-  /**
-   * @param {Object} [options]
-   * @param {String} [options.cacheRoot]
-   * @param {String} [options.stateRoot]
-   */
-  constructor ({ cacheRoot, stateRoot } = getDefaultRootDirs()) {
+  modules = [
+    'zinnia',
+    'saturn-L2-node',
+    'bacalhau'
+  ]
+
+  constructor ({ cacheRoot, stateRoot }) {
```

Comment:
Should we add a JSDoc comment to make the constructor private? We would need to inspect the `.d.ts` files produced by `tsc` to see if this has the desired effect of preventing people from calling the constructor directly.

```suggestion
  /** @private */
  constructor ({ cacheRoot, stateRoot }) {
```

Reply:
That works indeed! :)